### DISCUSSION TITLE
How can I get this telescope theme which found in wiki/What's new?

### DISCUSSION THREAD

**Discussion by JuanZoran** [folke/noice.nvim#404] [Q&A]
![image](https://user-images.githubusercontent.com/107862700/226605381-5d9dfac3-3200-4977-8b28-042f2312b5e5.png)
I found this picture from wiki,  but I can't found any way to setup this theme for telescope...

Maybe I should try to open a discussion in Telescope.nvim instead of noice.nvim

**Comment by folke** [comment]
That's not noice.

Check:

https://github.com/folke/dot/blob/master/nvim/lua/plugins/colorscheme.lua
https://github.com/folke/dot/blob/master/nvim/lua/plugins/telescope.lua

**Reply by JuanZoran** [reply]
Thanks a lot, folke!!! : )
